codecov_cli/services/upload/coverage_file_finder.py

```diff
@@ -58,7 +58,7 @@
     "*.exe",
     "*.ftl",
     "*.gif",
-    ".gitignore",
+    ".git*",
```

Comment:
ensures that .gitmodules, .gitattributes, and .gitignore are all exlcuded instead of just .git